auth0/testdata/recordings/TestAccConnectionApple.yaml

```diff
@@ -0,0 +1,172 @@
+---
+version: 1
+interactions:
+- request:
+    body: |
+      {"name":"Acceptance-Test-Apple-TestAccConnectionApple","strategy":"apple","is_domain_connection":false,"options":{"client_id":"client_id","app_secret":"-----BEGIN PRIVATE KEY-----\nMIHBAgEAMA0GCSqGSIb3DQEBAQUABIGsMIGpAgEAAiEA3+luhVHxSJ8cv3VNzQDP\nEL6BPs7FjBq4oro0MWM+QJMCAwEAAQIgWbq6/pRK4/ZXV+ZTSj7zuxsWZuK5i3ET\nfR2TCEkZR3kCEQD2ElqDr/pY5aHA++9HioY9AhEA6PIxC1c/K3gJqu+K+EsfDwIQ\nG5MS8Y7Wzv9skOOqfKnZQQIQdG24vaZZ2GwiyOD5YKiLWQIQYNtrb3j0BWsT4LI+\nN9+l1g==\n-----END PRIVATE KEY-----","team_id":"team_id","kid":"key_id","name":true,"email":true,"set_user_root_attributes":"on_each_login","non_persistent_attrs":null}}
```

Comment:
Secret is coming from https://github.com/auth0/terraform-provider-auth0/blob/f88e8e05978135bb9ead62ba84dec10873439a8e/auth0/resource_auth0_connection_test.go#L941